### PR TITLE
CI(github-actions): Use versioned tags for winget-releaser

### PR DESCRIPTION
**Pull request by vedantmgoyal2009**
### Checks

- [x] My commits follow the [commit guidelines](https://github.com/mumble-voip/mumble/blob/master/COMMIT_GUIDELINES.md)

### CODE REVIEW

**Review by davidebeatrici — APPROVED** [review]

**Comment by Krzmbrzl** [comment]
Your commit does not follow our commit guidelines. Your PR's title does, but jot your commit. Mind adapting it?

**Comment by vedantmgoyal2009** [comment]
@Krzmbrzl done 🙂

**Comment by Krzmbrzl** [comment]
Thanks :+1:

**Review by Krzmbrzl — APPROVED** [review]

File: .github/workflows/winget.yml [hunk 1]
```diff
@@ -7,18 +7,14 @@ jobs:
...
     steps:
       - name: Publish Mumble client
-        uses: vedantmgoyal2009/winget-releaser@latest
+        uses: vedantmgoyal2009/winget-releaser@v1
         with:
           identifier: Mumble.Mumble.Client
           installers-regex: mumble_client.*.msi$
           token: ${{ secrets.WINGET_TOKEN }}
 
-      # The action will clone winget-pkgs again, to start fresh
```
[the remark this answers is not in the document]
Comment (vedantmgoyal2009) [reply .github/workflows/winget.yml:16]: > I assume removing this extra step in the workflow is intentional?

I've added an extra step in the action to remove winget-pkgs folder if it exists before proceeding further. You can say that this step is inbuilt in the action itself.

**Pull request merged**